src/status_im/router/core.cljs

```diff
@@ -87,6 +92,22 @@
     {:type  :public-chat
      :error :invalid-topic}))
 
+(defn match-group-chat [uri _]
+  ;;TODO there is a bug in bidi, so we have to parse uri manually here
+  (let [[_ params] (when (not (string/blank? uri)) (string/split uri #"/g/"))
+        [admin-pk encoded-chat-name chat-id]  (when (not (string/blank? params)) (string/split params #"&"))
```

Comment:
This looks to me like not a standard Query string. I think using query params for that could be easier for multiple clients. Like `?admin-pk=0x0&encoded-chat-name=abcd&chat-id=abcd` https://en.wikipedia.org/wiki/Query_string#Structure

Also bidi does not have problems with standard query params.

Also, Query Strings may be easier to handle on `join.status.im`
cc. @jakubgs

Reply:
What are you asking me though?

Reply:
The current link format is using `&` to separate list of params.
```
https://join.status.im/g/0x0454a896b7b3307e402fa1ae78cb5c5dda27346f63c1581bbdc91eb3bed76253f367801a4fa44ee5ebd48c5e5210b69faea5671a26568e6fb2138fe6f1c4fb4bec&hi&63f5c71a-86b8-492a-a215-3dcca72fddbb-0x0454a896b7b3307e402fa1ae78cb5c5dda27346f63c1581bbdc91eb3bed76253f367801a4fa44ee5ebd48c5e5210b69faea5671a26568e6fb2138fe6f1c4fb4bec
```
While I was thinkig on this format.
```
https://join.status.im/g?admin-k=0x0454a896b7b3307e402fa1ae78cb5c5dda27346f63c1581bbdc91eb3bed76253f367801a4fa44ee5ebd48c5e5210b69faea5671a26568e6fb2138fe6f1c4fb4bec&chat-name=hi&chat-id=63f5c71a-86b8-492a-a215-3dcca72fddbb-0x0454a896b7b3307e402fa1ae78cb5c5dda27346f63c1581bbdc91eb3bed76253f367801a4fa44ee5ebd48c5e5210b69faea5671a26568e6fb2138fe6f1c4fb4bec
```

Asking you cause I know that you do the changes on unviersal link websites usually

Reply:
Damn that's a long-ass URL.

I'd say using proper URL arguments is cleaner, though wondering if some part of it should be part of the path?

Though I'm kinda confused. I thought referral program arguments will be passed via URL arguments for normal channel/chat links, like:
```
https://join.status.im/some-channel-name?some-referral-garbage=1231231231231231231231231231231232131
```
or
```
https://join.status.im/u/some-username?some-referral-garbage=1231231231231231231231231231231232131
```
But now you're showing me something completely new, what is `/g` supposed to be?

Reply:
This PR is about Group Chats invite (By this link user can join a group chat), not related to referrals

Reply:
> though wondering if some part of it should be part of the path
Yeah, I think maybe `chat-id` could be a route param

Reply:
yeah i thought about that as well, will change it to query params

Reply:
I think that's better, it makes the most important part of the URL part of the path, and the rest be parameters.